pytest/tests/loadtest/loadtest.py

```diff
@@ -11,63 +10,81 @@
     parser = argparse.ArgumentParser(description='Setup loadtest')
 
     parser.add_argument('--home', type=str, required=True)
+    # File in home directory that contains the key for the account that should be used to send all the requests.
+    parser.add_argument('--sender_key_file',
+                        type=str,
+                        default="validator_key.json")
```

Comment:
`add_argument` has a `help` keyword argument where you can put the help message instead of adding the comment.  It will then be visible in `--help` screen.

Reply:
good idea -done.